cabinets/cabinet/__init__.py

```diff
@@ -38,18 +40,28 @@ def set_configuration(cls, **kwargs):
         pass  # pragma: no cover
 
     @classmethod
-    def read(cls, path, raw=False, **kwargs):
-        if raw:
-            return cls.read_content(path)
-        else:
-            return Parser.load(path, cls.read_content(path), **kwargs)
+    def read(cls, path, parser: Union[bool, Type[Parser]] = True, **kwargs):
+        if parser is True:
+            return Parser.load(path, cls.read_content(path, **kwargs), **kwargs)
```

Comment:
Do it actually make sense to pass the same `kwargs` to both of these functions?

Reply:
I think it's useful to have `kwargs` for both. It's not great to pass the same ones to both, but I don't see a great way to specify `kwargs` separately for each. The only real issues here would come from duplicate argument names in the `Parser` and `Cabinet`.

Reply:
Yeah I mean that add a fairly odd constraint into the package that would be very easy to miss. A couple of ideas:
1. The user could pass a **neseted** dictionary with top-level keys for reading and loading. You could also allow other top-level keys if you pop the special ones off first.
2. Check for a common prefix/suffix on each keyword argument so that if a duplicate key is needed it can be separated into the respective function.

Reply:
The first option feels like it would decrease usability of `kwargs` pretty significantly. The second option is pretty neat actually -- arguments with `cabinet_` or `parser_` would be passed only to those classes respectively.

Reply:
Yup, my thoughts exactly. Just thought I'd throw the first option to make the second one sound even cooler.